src/test/scala/org/scalameter/MethodInvocationCountTest.scala

```diff
@@ -0,0 +1,7 @@
+package org.scalameter
+
+class MethodInvocationCountTest extends InvocationCountMeasurerTest {
+  test("MethodInvocationCountTest.allocations should be deterministic") {
+    checkInvocationCountMeasurerTest(new org.scalameter.examples.MethodInvocationCountTest1)
```

Comment:
I would do that - insert all the necessary keys early, in `prepareContext`, and then perform side-effects immediately before the tests start in `beforeContext.